css/style.css

```
@@ -0,0 +1,903 @@
+        padding-top: 60px
+    }
+    .main-information {
+        flex-direction: column;
+        align-items: center;
+        padding-top: 40px
+    }
+    .main-information__text {
+        width: 500px
+    }
+    .main-image__style {
+        width: 500px;
+        height: 300px;
+        margin-bottom: 40px
+    }
+    .featured-works {
+        padding-top: 80px
+    }
+    .featured-works__images {
+        margin-top: 80px
+    }
+    .featured-works__images img {
+        width: 260px
+    }
+    .popup__content img {
+        height: 50vh
+    }
+    .small-team__images {
+        margin-top: 80px
+    }
+    .contact {
+        padding-top: 50px
+    }
+    .contact-wrapper {
+        flex-direction: column;
+        align-items: center
+    }
+    .contact-form {
+        width: 520px;
+        margin-bottom: 15px
+    }
+    .contact-contact {
+        width: 520px;
+        margin-bottom: 50px
+    }
+    .map iframe {
+        width: 475px
+    }
+}
+
+@media (max-width:600px) {
+    .container {
+        width: 100%;
+        padding: 0 10px
+    }
+    .header {
+        height: 530px
+    }
+    .header_text__style {
+        margin-top: 50px;
+        margin-bottom: 50px
+    }
+    .main-information {
+        padding-top: 40px
+    }
+    .main-information__text {
+        width: 300px
+    }
+    .first-par,
+    .second-par {
+        text-align: center;
+    }
+    .main-image__style {
+        width: 300px;
+        height: 300px;
+        margin-bottom: 40px
+    }
+    .featured-works {
+        padding-top: 50px
+    }
+    .featured-works .featured-works__text {
+        line-height: 35px
+    }
+    .featured-works__images {
+        margin-top: 50px;
+        margin-bottom: 50px
+    }
+    .featured-works__images img {
+        width: 100%
+    }
+    .popup__content img {
+        height: 40vh
+    }
+    .small-team {
+        padding-top: 50px;
+        padding-bottom: 50px
+    }
+    .small-team__images {
+        margin-top: 50px
+    }
+    .team-member-first {
+        width: 300px;
+        background-size: cover;
+        background-position: center;
+        background-repeat: no-repeat;
+    }
+    .team-member_second {
+        width: 300px;
+        background-size: cover;
+        background-position: center;
+        background-repeat: no-repeat;
+    }
+    .team-member_third {
+        width: 300px;
+        background-size: cover;
+        background-position: center;
+        background-repeat: no-repeat;
+    }
+    .team-member_four {
+        width: 300px;
+        background-size: cover;
+        background-position: center;
+        background-repeat: no-repeat;
+    }
+    .team-member_five {
+        width: 300px;
+        background-size: cover;
+        background-position: center;
+        background-repeat: no-repeat;
+    }
+    .team-member_six {
+        width: 300px;
+        background-size: cover;
+        background-position: center;
+        background-repeat: no-repeat;

```

this can be `css variable`

---

Thanks, used css variable
```suggestion
width: var(--primary-width);
```